tests/foreman/cli/test_ldapauthsource.py

```diff
@@ -127,7 +127,7 @@ def test_positive_refresh_usergroup_with_ad(self, member_group, ad_data):
                 'attr-firstname': LDAP_ATTR['firstname'],
                 'attr-lastname': LDAP_ATTR['surname'],
                 'attr-mail': LDAP_ATTR['mail'],
-                'account': ad_data['ldap_user_name'],
+                'account': fr"{ad_data['workgroup']}\{ad_data['ldap_user_name']}",
```

Comment:
Nitpick comment about flipping the strings :)
```suggestion
                'account': fr'{ad_data["workgroup"]}\{ad_data["ldap_user_name"]}',
```